samples/concepts.js

```diff
@@ -1018,11 +1018,11 @@ class Transaction extends TestHelper {
       // Restore `datastore` to the mock API.
       datastore = datastoreMock;
       assert.strictEqual(
-        accounts[0].balance,
+        accounts[0].balance.valueOf(),
```

Comment:
Could we add a couple examples in:

https://github.com/googleapis/nodejs-datastore/blob/master/samples/concepts.js#L272

That demonstrate:

1. how to save an entry with an integer value (I believe we already have this).
2. how to fetch an entry with an integer value.
3. potentially how we'd JSON stringify an entry we fetched?

I know we're already showing the conversion with `accounts[0].balance.valueOf()`, but I think a more self contained example might be nice.

Reply:
I am not sure this can be done without looking odd. This sample exists alongside other language ones that are this simple.

I would be fine making a future work issue for this. 

/cc: @BenWhitehead

Reply:
Since we don't currently have any section in the docs discussing how to work with "large" numbers, and I'm not sure when we will. I think it'd be good to at least have a commented sample in the repo that can be discovered and referred to independent from this development PR. Once we have the sample we can add a work item to get it published into the docs.

Reply:
I am capturing this in an internal bug b/14309461.

If you feel strongly this is needed, it may belong in a doc. comment with the function being called?

Reply:
As long as there is something (either in samples, or in the reference docs alongside the code) so that someone can learn about this, my main concern is satisfied. Since this is a bit of an odd compatibility case that not all of the supported languages necessitate I think it'll take some design/work to incorporate into the docs based on how they're structured right now.